packages/app-elements/src/ui/atoms/ButtonCard.tsx

```diff
@@ -0,0 +1,95 @@
+import cn from 'classnames'
+import { Icon, type IconProps } from './Icon'
+
+type ButtonCardPadding = 'none' | '4' | '6'
+
+const paddingCss: Record<ButtonCardPadding, string> = {
+  none: 'p-0',
+  '4': 'p-4',
+  '6': 'p-6'
+}
```

Comment:
This is risky since these class names won't be generated by tailwind in this way. 
It's still fine since those rules are already included from other components, but it's risky in case we add a padding rule that does not exist elsewhere.

Reply:
Good observation.. Anyway I already tested this approach and it seems to be working.. At first there were more padding variants available.. Before moving to number string it was possibile to provide a full padding like `px-6 py-4` and so I tested that these padding sets defined in these record were correctly available..